mono/mini/interp/interp.c

```diff
@@ -3664,6 +3677,29 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, FrameClause
 
 			MINT_IN_BREAK;
 		}
+		MINT_IN_CASE(MINT_JIT_CALL2) {
+#ifdef ENABLE_EXPERIMENT_TIERED
+			InterpMethod *rmethod = (InterpMethod *) READ64 (ip + 1);
+
+			MONO_API_ERROR_INIT (error);
+			frame->ip = ip;
+
+			sp = do_jit_call (sp, vt_sp, context, frame, rmethod, error);
+			if (!is_ok (error)) {
+				MonoException *ex = mono_error_convert_to_exception (error);
+				THROW_EX (ex, ip);
+			}
+			ip += 5;
+
+			CHECK_RESUME_STATE (context);
+
+			if (rmethod->rtype->type != MONO_TYPE_VOID)
+				sp++;
+#else
+			g_assert_not_reached ();
```

Comment:
I don't like this macro because it generates generic message. Would be nicer to say `g_error ("something")` imho, but/and avoid varargs.